content/news/2022-03-28-release-22.03-highlights.md

```diff
@@ -1,13 +1,16 @@
 +++
 title = "Release 22.03 Highlights"
 date = 2022-03-28T16:25:00Z
+updated = 2022-04-04T13:25:00Z
 type = "post"
 description = "Highlights of the 22.03 release: the health-check flag, experimental DAP support, improved incremental injection parsing, and an overhauled tree-sitter grammar system."
 in_search_index = true
 +++
 
 Ranging from small quality-of-life improvements and fixes to large features
-and refactors, Helix 22.03 brings some exciting changes.
+and refactors, Helix 22.03 brings some exciting changes. Helix is a modal
+text editor with built-in support for multiple selections, LSP, tree-sitter,
+and now DAP.
```

Comment:
```suggestion
text editor with built-in support for multiple selections, Language
Server Protocol (LSP), tree-sitter, and now Debug Adapter Protocol (DAP).
```
I think would be better to show out the name so user can at least guest the meaning without searching for it.